server/app/test_bids_payments.py

```diff
@@ -0,0 +1,175 @@
+# This is a draft test before ERM review finished;
+# Test CRUD on table Bids and Payments;
+# Assume user and task are already created with no problems before testing Bids and Payments.
+from django.test import TestCase
+from django.core.exceptions import ValidationError
+from .models import Users, Tasks, Bids, Payments
+from datetime import datetime, timedelta
+
+class BidsPaymentsTestCase(TestCase):
+
+    def setUp(self):
+        # Create a test user
+        self.user = Users.objects.create(
+            email="testuser@example.com",
+            mobile="1234567890",
+            password_hash="hashed_password",
```

Comment:
Instead of using a plain string like "hashed_password" for password_hash, you should use Django's built-in hash generation method. For example, use make_password("secure_password") to properly hash the password in the test setup.
@HuxleyBerry

Reply:
Thanks. added make_password and import module django.contrib.auth.hashers